pkg/kubelet/kubelet_pods.go

```diff
@@ -32,7 +32,7 @@ import (
 	"strings"
 	"sync"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: unrelated change

Reply:
This is automatically modified by one of the go formatting tools. Many files in Kubernetes have this now.